### ISSUE TITLE
feat: Adapt to the current timezone.

### ISSUE THREAD

**Issue by Sknp1006** [YunYouJun/valaxy#229]
### Clear and concise description of the problem

When using the Valaxy VSCode plugin preview, I noticed that the article's time is in the UTC Time. It may be necessary to adapt it to the localization of the time zone. 

### Suggested solution

I believe you can solve it.

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by zmxlt** [comment]
Yes, the default time zone is utc +0, so I use +8 manually

```
---
date: 2022-04-09 +8
---
```
```
---
date: 2023-04-09 08:00:00 +8
---
```

**Comment by Sknp1006** [comment]
wow,that‘s a useful tip,it works.

**Comment by YunYouJun** [comment]
Maybe we need add a config for timezone?

**Comment by YunYouJun** [comment]
`timezone` had been added in `siteConfig`. Released in v0.14.32.
